newscoop.conf

```diff
@@ -0,0 +1,13 @@
+<VirtualHost *:80>
+      DocumentRoot /var/www/newscoop
+      ServerName docker.newscoop
```

Comment:
let's set it to `localhost` This will not require adding vhost on the user's machine to open docker instance and it will fix the issue running docker on OSX when you need to type VM address. If its set like that for docker on mac then when entering docker instance, you will see default apache page.